scss/_variables.scss

```
@@ -1643,4 +1643,29 @@ $back-to-top-icon:               var(--#{$boosted-variable-prefix}chevron-icon)
 $back-to-top-icon-width:         add(.5rem, 1px) !default;
 $back-to-top-icon-height:        subtract(1rem, 1px) !default;
 // scss-docs-end back-to-top
+
+//// Stepped process
+// scss-docs-start stepped-process
+$stepped-process-max-items: 5 !default;
+$stepped-process-counter:   step !default; // Used as a counter name
+
+$step-item-max-width:    3.75rem !default;
+$step-item-margin-end:   $border-width !default;
+$step-item-padding:      $spacer / 2 !default;
+$step-item-bg:           $black !default;
+$step-item-active-bg:    $primary !default;
+$step-item-next-bg:      $gray-400 !default;
+$step-item-shadow-size:  $border-width * 1.5 !default;
+$step-item-drop-shadow:  drop-shadow($step-item-shadow-size 0 0 $white) #{"/* rtl:"} drop-shadow(-$step-item-shadow-size 0 0 $white) #{"*/"} !default;
```

Simpler version, needs to check if that works:
```suggestion
$step-item-drop-shadow:  drop-shadow(#{"/*rtl:insert:-*/"}$step-item-shadow-size 0 0 $white) !default;
```

We're using this insertion in `$font-family-sans-serif` in `_variables.scss` for example. Useful when you don't need to change the whole value but only to add something in the existing one.

---

That was a good idea, except it seems sass adds a space, breaking the rtl version. Closing :(